Update LUMI web interface docs

robinkar opened the pull request:
Updates outdated images and documentation to match what is currently in LUMI web interface release 8.

Main changes:
- Jupyter docs (container usage).
- Julia-Jupyter docs reworked to be more consistent with other pages.
- Desktop apps have been added.
- Outdated images updated to match the currently deployed version.

robinkar committed:
Update LUMI web interface docs

Update outdated images and documentation to match what is currently in
LUMI web interface release 8.

hdrei commented on docs/runjobs/webui/jupyter.md:
```diff
@@ -7,34 +7,58 @@ In the form you will be able to select:
+    `jupyterlab` package has been installed. Jupyter will be launched using the
+    command `python -m jupyter lab`.
+- **Container:**
+    Allows you to use a containerized Python installation. You can can specify
```
```suggestion
    Allows you to use a containerized Python installation. You can specify
```

hdrei commented on docs/runjobs/webui/julia-jupyter.md:
```diff
@@ -1,35 +1,27 @@
-Next, we must set the desired computing resources: CPU cores, memory, local disk, and time.
-
-![julia jupyter options](../../assets/images/julia-jupyter/options-2.png)
+In the form, start by selecting the project for billing, partition for computing resources,
+and the desired computing resources: CPU cores, memory, local disk, and time.
```
```suggestion
In the form, start by selecting the project for billing, partition,
and the desired computing resources: CPU cores and time.
```

hdrei replied:
It doesn't seem to be possible to select "memory" and "local disk" anymore

robinkar replied:
The memory dropdown appears on partitions where the memory does not depend on the number of CPU cores, e.g. on the `small` partition. Local disk would appear if the `lumid` partition is selected, but is currently disabled in the web interface.

hdrei commented on docs/runjobs/webui/desktop.md:
```diff
@@ -24,16 +24,27 @@ Once the job has started, there are two ways to connect to the desktop.
+
+### GPU Acceleration
+
+All of the available apps will utilize a GPU for rendering if the _lumid_
+partition was selected. However, note that not all of the applications benefit
+from having a GPU available.
```
I'm not sure, if this should be clarified somehow.
Is it clear for the user, if an application benefits "from having a GPU available"?
Should it be advised to use some other partition in case the application doesn't benefit from the GPU?

robinkar replied:
Benefit highly depends on the application and data/workload, but means higher frame rate in the application.
Clarified the section a bit to recommend using the CPU partition if it is good enough.

robinkar committed:
Clarify details in LUMI web interface docs

robinkar committed:
Rework Julia-Jupyter form option docs

Remove parts that are generic for all interactive apps. Add a list for
the rest to make it more consistent with the normal Jupyter docs.
